recipes/rank_filter/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "rank_filter" %}
+{% set version = "0.4.11" %}
+{% set checksum = "246f5c73ebd03314d9b2cff3ede74330" %}
```

Comment:
Just saw that and commented. :+1: for being inspired by our collaborators - that is what this is all about :+1: 

Let's resolve that in one place, and we can move forwards from there.